src/java.base/unix/native/libnio/fs/UnixCopyFile.c

```diff
@@ -87,7 +120,12 @@ Java_sun_nio_fs_UnixCopyFile_transfer
     do {
         RESTARTABLE(sendfile64(dst, src, NULL, count), bytes_sent);
         if (bytes_sent == -1) {
-            throwUnixException(env, errno);
+            if (errno == EINVAL) {
```

Comment:
Don't you think we have have to take care of "ENOSYS" as well. Manual pages talks about "ENOSYS" as well.
 
"Applications may wish to fall back to read(2)/write(2) in the case where sendfile() fails with EINVAL or ENOSYS."

Reply:
I think that's a good point.